vividus-extension-selenium/src/main/java/org/vividus/steps/ui/validation/BaseValidations.java

```diff
@@ -264,4 +273,20 @@ public List<WebElement> assertIfNumberOfElementsFound(String businessDescription
                     elementNumber(comparisonRule.getComparisonRule(number))) ? elements : List.<WebElement>of();
         }).orElseGet(List::of);
     }
+
+    private String getElementFoundAsserionMsg(String description, Locator locator, boolean found)
+    {
+        StringBuilder assertionMessageBuilder = new StringBuilder(description).append(" is ");
+        if (!found)
+        {
+            assertionMessageBuilder.append("not ");
+        }
+        return assertionMessageBuilder.append("found by the locator ").append(locator).toString();
```

Comment:
```suggestion
        return assertionMessageBuilder.append("found by the locator ").append(locator.toHumanReadableString()).toString();
```

but keep in mind this method can't be used in old methods